### PR TITLE
Importer env

### PR DESCRIPTION
**Pull request by luigibk**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-874

### Change description ###
Move all ccd importer parameters to environment variables that can be passed from docker file (or kubernetes definition at some point).

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -192,6 +192,15 @@ services:
...
         VERBOSE: ${VERBOSE:-false}
         WAIT_HOSTS_TIMEOUT: 300
+        CREATE_IMPORTER_USER: "yes"
+        IMPORTER_USERNAME: ccd-importer@server.net
+        IMPORTER_PASSWORD: Password12
+        IDAM_URI: http://idam-api:8080
+        REDIRECT_URI: http://localhost:3000/receiver
+        CLIENT_ID: bsp
+        CLIENT_SECRET: 123456
+        CCD_ROLE: caseworker-bulkscan
```
Comment (timja) [inline docker-compose.yml]: role or roles?

File: docker-compose.yml [hunk 2]
```diff
@@ -192,6 +192,15 @@ services:
...
+        BULK_SCAN_ORCHESTRATOR_BASE_URL: http://host.docker.internal:8582
```
Comment (timja) [inline docker-compose.yml]: should this have bulk_scan in it?

File: docker/ccd-definition-import/scripts/upload-definition.sh [hunk 3]
```diff
@@ -3,25 +3,18 @@ set -e
...
-BULK_SCAN_ORCHESTRATOR_BASE_URL=http://host.docker.internal:8582
-
 if [ ${VERBOSE} = "true" ]; then
   export CURL_OPTS="-v"
 else
   export CURL_OPTS="--fail --silent"
 fi
 
-/scripts/create-importer-user.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD}
+[ "_${CREATE_IMPORTER_USER}" = "_yes" ] && /scripts/create-importer-user.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD}
```
Comment (timja) [inline docker/ccd-definition-import/scripts/upload-definition.sh]: why the `_`

File: docker-compose.yml (hunk 2 above)
Comment (luigibk) [reply docker-compose.yml]: Default values should work locally

File: docker-compose.yml (hunk 1 above)
Comment (luigibk) [reply docker-compose.yml]: role currently

File: docker-compose.yml (hunk 2 above)
Comment (timja) [reply docker-compose.yml]: I mean should the environment variable be called BULK_SCAN_ORCHESTRATOR or BASE_URL?

File: docker/ccd-definition-import/scripts/upload-definition.sh (hunk 3 above)
Comment (luigibk) [reply docker/ccd-definition-import/scripts/upload-definition.sh]: Precaution in case the variable is not set at all (not to get an ugly error message).

**Pull request merged**